src/core/api/public_api.ts

```diff
@@ -1109,6 +1110,30 @@ class Player extends EventEmitter<IPublicAPIEvent> {
     return this.state;
   }
 
+  /**
+   * Returns true if a content is loaded.
+   * @returns {Boolean} - `true` if a content is loaded, `false` otherwise.
+   */
+  isContentLoaded() : boolean {
+    return !arrayIncludes(["LOADING", "RELOADING", "STOPPED"], this.state);
+  }
+
+  /**
+   * Returns true if the player is rebuffering.
+   * @returns {Boolean} - `true` if the player is rebuffering, `false` otherwise.
+   */
+  isRebuffering() : boolean {
+    return arrayIncludes(["BUFFERING", "SEEKING", "LOADING", "RELOADING"], this.state);
+  }
+
+  /**
+   * Returns true if the content is not currently playing.
+   * @returns {Boolean} - `true` if the media is paused, `false` otherwise.
+   */
+  isPaused() : boolean {
+    return !arrayIncludes(["PLAYING", "STOPPED"], this.state);
```

Comment:
So you see it as only `return this.videoElement ? this.videoElement.paused : true;`?

Reply:
Yeah why not?

To note that `videoElement` is only unset when the RxPlayer is disposed (its `dispose` method has been called), which leads to the impossibility to continue using it (the point of this method is to free its resources, some of them being kept alive because linked to the media element linked to the RxPlayer).
So we could also be throwing or do whatever, but I guess returning `true` is the more resilient solution

Reply:
Ok great!